CP05MOAS-GL339/CP05MOAS-GL339_R00014_ingest.csv

```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00014/merged/cp_*.mrg,CP05MOAS-GL339-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00014/dvl/*.PD0,CP05MOAS-GL339-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00014/merged/cp_*.mrg,CP05MOAS-GL339-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00014/merged/cp_*.mrg,CP05MOAS-GL339-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00014/merged/cp_*.mrg,CP05MOAS-GL339-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/R00014/merged/cp_*.mrg,CP05MOAS-GL339-05-PARADM000,recovered_host,Available,
```

Comment:
looks good - data available

Reply:
DVL disabled for part of the deployment? Otherwise OK

Reply:
update looks good (per PB's comment)